src/hook_files/hook_script.sh

```diff
@@ -21,5 +21,6 @@ else
   ensureMinimumRustyHookCliVersion || true
 fi
 
-rusty-hook run --hook "${hookName}" "${gitParams}"
+# shellcheck disable=SC2046
+rusty-hook run --hook "${hookName}" $([ -z "$gitParams" ] && echo "-- $gitParams" || echo "")
```

Comment:
Sorry I didn't look at this closely enough yesterday, but aren't the conditions reversed? Wouldn't this only try to add the `-- $gitParams` value when it's empty? 

```suggestion
rusty-hook run --hook "${hookName}" $([ -z "$gitParams" ] && echo "" || echo "-- $gitParams")
```

Reply:
Whoops, done!